models/comments.models.js

```diff
@@ -1,10 +1,17 @@
 const db = require("../db/connection");
+const { selectArticle } = require("./articles.models");
 
 exports.selectCommentsByArticleId = (article_id) => {
     const sql = `SELECT * FROM comments WHERE article_id = $1 
     ORDER BY created_at DESC;`;
 
-    return db.query(sql, [article_id]).then( ({rows}) => {
+    return selectArticle(article_id).then(({rows}) => {
```

Comment:
This is one way to handle this. In other servers you may find it useful to extract out a function that checks if a particular row exists in any table. But if your tests are passing, nicely done.

Reply:
Good point having a utility function has its advantages. It wouldn't reduce the number of queries though would it? I guess creating a ref object/ array could be something to try later.

Reply:
It is the same number of api calls but it is helpful to have a function where confirming an article exisits is its specific purpose to allow for for clarity and control.